### PR TITLE
	RavenDB-7490 Implement a way to delete all revisions for document id

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/VersioningHandler.cs [hunk 1]
```diff
@@ -179,5 +180,40 @@ public Task GetZombiedRevisions()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/revisions", "DELETE", "/databases/*/revisions?id={documentId:string|multiple}")]
+        public async Task DeleteRevisionsFor()
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/VersioningHandler.cs:185]: This needs to be an admin only operation, I think

File: src/Raven.Server/Documents/Versioning/VersioningStorage.cs [hunk 2]
```diff
@@ -323,6 +323,32 @@ private VersioningConfigurationCollection GetVersioningConfiguration(CollectionN
             IncrementCountOfRevisions(context, prefixSlice, -deletedRevisionsCount);
         }
 
+        public void DeleteRevisionsFor(DocumentsOperationContext context, string id)
+        {
+            using (DocumentIdWorker.GetSliceFromId(context, id, out Slice lowerId))
+            using (GetKeyPrefix(context, lowerId, out Slice prefixSlice))
+            {
```
Comment (ayende) [inline src/Raven.Server/Documents/Versioning/VersioningStorage.cs:330]: Add logging here.

Comment (fitzchak) [reply src/Raven.Server/Documents/Versioning/VersioningStorage.cs:330]: Done

File: src/Raven.Server/Documents/Handlers/VersioningHandler.cs (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Server/Documents/Handlers/VersioningHandler.cs:185]: Done

**Pull request merged**